Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CylindricalYeeAlgorithm.H

```diff
@@ -33,6 +35,29 @@ struct CylindricalYeeAlgorithm {
         stencil_coefs_z[0] = 1._rt/cell_size[2];  // 1./dz
     }
 
+    /**
+     * Compute the maximum timestep, for which the scheme remains stable
+     * (Courant-Friedrichs-Levy limit) */
+    static amrex::Real ComputeMaxDt ( amrex::Real const * const dx,
+                                      int const n_rz_azimuthal_modes ) {
+        // In the rz case, the Courant limit has been evaluated
+        // semi-analytically by R. Lehe, and resulted in the following
+        // coefficients.
+        amrex::Real multimode_coeffs[6] = { 0.2105, 1.0, 3.5234, 8.5104, 15.5059, 24.5037 };
+        amrex::Real multimode_alpha;
+        if (n_rz_azimuthal_modes < 7) {
```

Comment:
I would potentially place the `6` in a `constexpr` and use it here and in the definition of the constants.